templates/myaccount/form-edit-account.php

```diff
@@ -35,6 +35,12 @@
 		<input type="text" class="woocommerce-Input woocommerce-Input--text input-text" name="account_last_name" id="account_last_name" value="<?php echo esc_attr( $user->last_name ); ?>" />
 	</p>
 	<div class="clear"></div>
+	
+	<p class="woocommerce-form-row woocommerce-form-row--wide form-row form-row-wide">
+		<label for="account_display_name"><?php esc_html_e( 'Display name publicly as', 'woocommerce' ); ?> <span class="required">*</span></label>
```

Comment:
Leave as display name, but add a description below the field e.g. 'This will be how your name will be displayed in the account section and in reviews'.

Reply:
Thanks for all your feedbacks.